testnets/chain4energytestnet/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "chain4energy",
```

Comment:
please change to 'chain4energytestnet' (one word, to match the folder name exactly)

Reply:
done